src/shaping/shaping_linux.go

```diff
@@ -168,7 +167,7 @@ func shape_on(id int64, shaping *atc_thrift.LinkShaping, link netlink.Link) erro
 	// rate was not set and is considered unlimited.
 	// In that case, let set the rate as high as we can.
 	// Note: currently, it is implemented as a u32 by the netlink library.
-	rate := uint64(shaping.GetRate() * 1000)
+	rate := uint64(shaping.Rate * 1000)
```

Comment:
It's a change in how thrift generates golang code. GetProperty() is way less idiomatic than .Property. I am using an older version of the thrift-compiler (0.9.3 vs 0.9.1) but the thrift library that provides the golang support and generation code is much newer.